portfolio/src/main/webapp/script.js

```diff
@@ -42,10 +42,53 @@ function fadeOut(){
 }
 
 //From Week-3 Tutorial
-function getGreeting() {
-  console.log('Handling the response.');
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementsByClassName('greeting-cont')[0].innerText = quote;
+function getComments() {
+  numComments = localStorage.getItem("num-of-comments");
+  console.log(numComments);
+  fetch(`/comments?num-comments=${numComments}`).then(response => response.json()).then((comment) => {
+    console.log(comment);
```

Comment:
consider removing the console.log throughout the code if we just used it for debugging. Otherwise, consider adding more contextual information to the logging.

Reply:
done